src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -35,11 +35,15 @@ public Drivetrain() {
     // left_master_talon.setInverted(true);
     // left_slave_talon.setInverted(true);
 
+    left_master_talon.configClosedloopRamp(Constants.DRIVETRAIN_RAMP);
+    left_slave_talon.configClosedloopRamp(Constants.DRIVETRAIN_RAMP);
+    right_master_talon.configClosedloopRamp(Constants.DRIVETRAIN_RAMP);
+    right_slave_talon.configClosedloopRamp(Constants.DRIVETRAIN_RAMP);
+
```

Comment:
This is really good, but since the slave talons follow the other talons they do not need this constant to be set.